services/data/src/types.ts

```diff
@@ -13,9 +13,7 @@ export interface ContextInput {
     apiVersion: number
 }
 
-export interface RefetchOptions {
-    variables?: QueryVariables
-}
+export type RefetchOptions = QueryVariables
```

Comment:
Wouldn't this cause another breaking change if we ever want to add additional arguments to the refetch options (e. g. network policies, cache options, etc)?

Reply:
Yeah this type was incorrect though. The changed code reflects the actual situation as it was before my changes.